packages/reactive-element/src/reactive-element.ts

```diff
@@ -676,6 +686,12 @@ export abstract class ReactiveElement
       (options as Mutable<PropertyDeclaration, 'attribute'>).attribute = false;
     }
     this.__prepare();
+    // Whether this property is wrapping accessors.
+    // Helps control the initial value change and reflection logic.
+    if (this.prototype.hasOwnProperty(name)) {
+      options = Object.create(options);
+      options.wrapped = true;
```

Comment:
Do we care about preserving the prototype relationship here? I would normally do:
```suggestion
      options = {...options, wrapped: true};
```

Reply:
There was a test that covered this case that we weren't previously exercising through here.